app/templates/events/view/tickets/attendees.hbs

```diff
@@ -0,0 +1,65 @@
+<div class="ui hidden divider"></div>
+<div class="ui compact menu pull right floated">
+  <div class="ui simple dropdown item">
+    {{t 'Showing all records'}}
+    <i class="dropdown icon"></i>
+    <div class="menu">
+      <div class="item">{{t 'Filter by Date'}}</div>
+      <div class="item">{{t 'Filter by Ticket'}}</div>
+      <div class="item">{{t 'Remove all Filters'}}</div>
+    </div>
+  </div>
+</div>
+<div class="ui compact menu pull right floated">
+  <div class="ui simple dropdown item">
+    {{t 'Export As'}}
+    <i class="dropdown icon"></i>
+    <div class="menu">
+      <div class="item">{{t 'PDF'}}</div>
+      <div class="item">{{t 'CSV'}}</div>
+    </div>
+  </div>
+</div>
+<div class="content">
+  <h2 class="ui header">{{t 'View Attendees'}}</h2>
+</div>
+<div class="ui hidden divider"></div>
+<div class= "content">
+  <div class="ui stackable basic buttons">
+    <div class="ui button active">{{t 'Completed'}}</div>
+    <div class="ui button">{{t 'Placed'}}</div>
+    <div class="ui button">{{t 'Pending'}}</div>
+    <div class="ui button">{{t 'Expired'}}</div>
+    <div class="ui button">{{t 'Cancelled'}}</div>
+    <div class="ui button">{{t 'Checked In'}}</div>
+    <div class="ui button">{{t 'Not Checked In'}}</div>
+    <div class="ui button">{{t 'All'}}</div>
+  </div>
+  <table class="ui celled table">
+    <thead>
+      <tr>
+        <th>#</th>
+        <th>{{t 'Order'}}</th>
+        <th>{{t 'Ticket Name'}}</th>
+        <th>{{t 'Ticket Price'}}</th>
+        <th>{{t 'First Name'}}</th>
+        <th>{{t 'Last Name'}}</th>
+        <th>{{t 'Email'}}</th>
+        <th>{{t 'Action'}}</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td>{{t 'No data available in Table'}}</td>
```

Comment:
Please fill this with actual data, use orga server for reference on so as to how the table should look with the data filled in.